ugali/preprocess/mangle.py

```diff
@@ -12,10 +12,10 @@ def fits2ply(infile,outfile=None):
         outfile = base + '.ply'
     if os.path.exists(outfile): os.remove(outfile)
 
-    print "Converting %s to %s"%(infile,outfile)
+    print("Converting %s to %s"%(infile,outfile))
 
     ### Python Routine
-    import mangle
+    from . import mangle
```

Comment:
Hmmm, I'm not sure about this one... This code is deprecated, so it probably doesn't matter.